Seohee/Lv2/교점에 별만들기.js

```diff
@@ -0,0 +1,37 @@
+function solution(line) {
+  // 교점의 최대/최소 좌표
+  let [maxX, minX, maxY, minY] = [-Infinity, Infinity, -Infinity, Infinity];
+  let points = [];
+  //교점 찾아서 넣기
+  for (let i = 0; i < line.length - 1; i += 1) {
+    for (let j = i + 1; j < line.length; j += 1) {
+      const denominator = line[i][0] * line[j][1] - line[i][1] * line[j][0];
```

Comment:
```suggestion
      const denominator = (line[i][0] * line[j][1]) - (line[i][1] * line[j][0]);
```
동작상에 문제는 없지만 괄호로 감싸주는게 가독성이 더 좋을거같아요.